src/internal/coredb/projects_test.go

```diff
@@ -81,27 +81,45 @@ func TestListProject(t *testing.T) {
 	migrationEnv := migrations.Env{EtcdClient: testetcd.NewEnv(ctx, t).EtcdClient}
 	require.NoError(t, migrations.ApplyMigrations(ctx, db, migrationEnv, clusterstate.DesiredClusterState), "should be able to set up tables")
 	require.NoError(t, DeleteProject(ctx, db, "default"))
-	expectedInfos := make([]*pfs.ProjectInfo, 100)
-	for i := 0; i < 100; i++ {
+	size := 210
+	expectedInfos := make([]*pfs.ProjectInfo, size)
+	for i := 0; i < size; i++ {
 		createInfo := &pfs.ProjectInfo{Project: &pfs.Project{Name: fmt.Sprintf("%s%d", testProj, i)}, Description: testProjDesc, CreatedAt: types.TimestampNow()}
 		expectedInfos[i] = createInfo
 		require.NoError(t, CreateProject(ctx, db, createInfo), "should be able to create project")
 	}
 	iter, err := ListProject(ctx, db)
 	require.NoError(t, err, "should be able to list projects")
-	defer iter.Close()
 	i := 0
-	for proj, err := iter.Next(); !errors.Is(err, io.EOF); proj, err = iter.Next() {
+	projectInfo := &pfs.ProjectInfo{}
+	for err := iter.Next(ctx, projectInfo); !errors.Is(err, io.EOF); err = iter.Next(ctx, projectInfo) {
```

Comment:
Nit: I know this is just test code, but lets be consistent
```suggestion
	for !errors.Is(iter.Next(ctx, projectInfo), err, io.EOF)  {
```